### ISSUE TITLE
Use python logging in the c extension

### ISSUE THREAD

**Issue by jacquemier** [cta-observatory/pyhessio#22]
This would make the output handling a lot better.

See here for how it can be done:
https://gist.github.com/hensing/0db3f8e3a99590006368

**Comment by mackaiver** [comment]
Hi,

would this help to get rid of these messages?

```
I/O block extended by 541484 to 3688356 bytes
```

**Comment by kosack** [comment]
Not really - those messages come from the underlying libhessio.so library, which we don't have access to modify.  There may be a verbosity setting for it though.

**Comment by kosack** [comment]
Yes, but we regularly sync this from the original hessioxxx repo, so it would be lost (and probably due to the GPL license we should contribute back any changes). Basically, libhessio is not ours to modify, we just include it for convenience. We could suggest to the developer (Konrad) that he support a less verbose mode.

**Comment by kosack** [comment]
have you looked at the `Information(msg)` function? It may already have an option to suppress or capture output
